docs/fidesops/docs/guides/saas_config.md

```diff
@@ -0,0 +1,370 @@
+## What is a SaaS configuration schema?
+
+A SaaS connector is defined in two parts, the [Dataset](datasets.md) and the SaaS config. The Dataset describes the data that is available from the connector and the SaaS config describes how to connect and retrieve/update the data in the connector. If you contrast this to a [database connector](database_connectors.md), the ways to retrieve/update data conform to a specification (such as SQL) and are consistent. When accessing data from APIs, each application or even different endpoints within the same application can follow different patterns. It was necessary to have a flexible configuration to be able to define the different access/update patterns. Keep in mind that SaaS configs are only applicable to SaaS connector, not database connectors.
```

Comment:
This would best be prefaced with a sentence giving an overview of what SaaS connectors do akin to "Fidesops allows users to access, mask and erase data from third party web services, in the form of SaaS connectors".

Reply:
Update: we have this here I see https://github.com/ethyca/fidesops/pull/259/files#diff-0a8628918d3aa2c3bff1b0686665a76c10b308d4641a24982fa162ad1b023745R3

Reply:
minor grammer: only applicable to SaaS connector(s)

Reply:
updated!